scipy/special/_basic.py

```diff
@@ -3020,8 +3020,8 @@ def factorial(n, exact=False):
             return math.factorial(n)
         elif exact:
             msg = ("Non-integer values of `n` together with `exact=True` are "
```

Comment:
```suggestion
            msg = ("Non-integer values of `n` together with `exact=True` is "
```

Reply:
I prefer this as-is (with "are", which refers to plural "values").

Reply:
Yes I agree, clearly too tired when I wrote that!